Sources/UIKit+Theme.swift

```diff
@@ -72,7 +72,7 @@ import UIKit
         get { return getThemePicker(self, "updateTextAttributes:") as? ThemeStringAttributesPicker }
         set { setThemePicker(self, "updateTextAttributes:", newValue) }
     }
-    var theme_attributesTexts: ThemeAttributedStringPicker? {
+    var theme_attributesText: ThemeAttributedStringPicker? {
```

Comment:
`theme_attributesText` => `theme_attributedText `

Change the letter 's' to 'd', please.

Reply:
Get it, Sorry about that.